pkgs/development/compilers/hare/hare/default.nix

```diff
@@ -4,23 +4,23 @@
 , binutils-unwrapped
 , harePackages
 , makeWrapper
-, qbe
+  #, qbe # We use harec's override of qbe until 1.2 is released
```

Comment:
```suggestion
```
comments in inputs are often overlooked

Reply:
I've moved the comment to the `qbe` variable in the `let-in` statement, while also keeping the `qbe` argument to avoid breakage with code that passes it to `hare`.